### PR TITLE
pass extra save_stream keywords to image2wand

### PR DESCRIPTION
**Pull request by johnnychen94**
Similar to the `save_` method for filename, this PR passes extra keywords to `image2wand`. An example of this is a stream gif saving with fps keyword. (https://github.com/JuliaImages/ImageShow.jl/pull/30/)

Also fixes a FileIO depwarn in the tests.

### CODE REVIEW

**Comment by johnnychen94** [comment]
The failure in ubuntu is reproducible locally and seems to be an SSL certificate issue.

```julia
julia> download("http://magnushoff.com/assets/test-exiforientation.zip")
┌ Error: Download failed: curl: (60) SSL certificate problem: unable to get local issuer certificate
└ @ Base download.jl:43
ERROR: failed process: Process(`curl -s -S -g -L -f -o /tmp/jl_3ekIQ8 http://magnushoff.com/assets/test-exiforientation.zip`, ProcessExited(60)) [60]
```

File: test/constructed_images.jl [hunk 1]
```diff
@@ -214,7 +214,7 @@ mutable struct TestType end
         orig_img = ImageMagick.load(joinpath(workdir, "2by2.png"))
         fn = joinpath(workdir, "2by2_fromstream.png")
         open(fn, "w") do f
-            ImageMagick.save(Stream(format"PNG", f), orig_img)
+            ImageMagick.save(Stream{format"PNG"}(f), orig_img)
```
Comment (timholy) [inline test/constructed_images.jl:217]: Interesting how much disruption this depwarn causes. Tests will fail unless FileIO >= 1.6, but I guess we can live with that, and it won't happen in CI or an update test environment.

**Review by timholy — APPROVED** [review]

**Comment by timholy** [comment]
> The failure in ubuntu is reproducible locally and seems to be an SSL certificate issue.

Interesting. Can we generate a "data artifact" and stop worrying about downloading this freshly each time? I haven't really followed that world very well.

**Comment by johnnychen94** [comment]
This file is only 4Kb (less than the 4kb block size) so I think we're good to commit it to the git?

**Comment by timholy** [comment]
That works for me unless there's some license/whatever reason that we can't.

**Pull request merged**